cmd/utils.go

```diff
@@ -831,3 +834,41 @@ func execAndWaitWithWorkDirReturnErr(command string, args []string, logger appso
 	}
 	return err
 }
+
+func checksum256TestFile(newFileName string, oldFileName string) (bool, error) {
+	var checkValue bool
+	fmt.Println("Checksum 256")
+	fmt.Println("oldFile", oldFileName)
+	fmt.Println("newFile", newFileName)
+
+	oldFile, err := os.Open(oldFileName)
+	if err != nil {
+		return false, errors.Errorf("File open failed for %s controller binary: %v", oldFileName, err)
```

Comment:
Perhaps log in a util function could be more generic and not refer to `controller binary`. We might be able to reuse this function for diff files in future